xml/install_caasp_heat_templates.xml

```diff
@@ -596,46 +504,32 @@ i | sles12-velum-image | package    | 3.1.7-3.27.3  | x86_64 | update_caasp</scr
       Upload a valid trust certificate that can validate a certificate
       presented by &o_ident; at the specified
       <literal>keystone_auth_url</literal> in the <literal>system-wide
-      certificate</literal> section of Velum. The certificate must be uploaded
-      to avoid bootstrapping failure with a <literal>x509: certificate signed
-      by unknown authority</literal> error message.
+      certificate</literal> section of Velum. If the SSL certificate provided
+      by &o_ident; cannot be verified, bootstrapping fails with the error
+      <literal>x509: certificate signed by unknown authority</literal> error
+      message.
```

Comment:
This repeats "error" twice. Can change to: "bootstrapping fails with a <literal>x509: certificate signed by unknown authority</literal> error message."